server/game/cards/01-Core/DanceOfDoom.js

```diff
@@ -6,7 +6,7 @@ class DanceOfDoom extends Card {
             target: {
                 activePromptTitle: 'Choose a number',
                 mode: 'options',
-                options: context => [... Array(Math.max.apply(Math, context.game.creaturesInPlay.map(card => card.power)) + 1).keys()].
+                options: context => [... Array(Math.max(0, ... context.game.creaturesInPlay.map(card => card.power)) + 1).keys()].
```

Comment:
I'm unclear on what the +1 does in this line now (Pretty unclear what it did before to be honest)

Reply:
The Array uses the number as length, so 4, produces [0,1,2,3], but we actually want [0,1,2,3,4], so we add 1. I added the 0, ... to force Math.max to have at lease 1 value.

It is a crazy syntax. I tried it many times to believe it actually did what I wanted it to do. The javascript "..." syntax is a mistery to me.